documentation/source/overview/concepts/warping-fields.rst

```diff
@@ -0,0 +1,45 @@
+
+.. code::
+
+   # Apply warping fields to an image file
+   sct_apply_transfo -i in.nii.gz -d out.nii.gz -w warp1.nii.gz warp2.nii.gz [...]
+   # Concatenate several warping fields into single warping field
+   sct_concat_transfo -d out_warp.nii.gz -w in_warp1.nii.gz in_warp2.nii.gz [...]
+
+Warping fields are used throughout SCT:
+
+* When straightening a spinal cord image, two warping fields are created, which define the forward and inverse transformations between the curved anatomical image and the straightened image.
+* When registering a spinal cord image to a template, two warping fields are created, which define the forward and inverse transformations between the anatomical image space and the template space.
+
+Warping field conventions
+=========================
+
+In the broader ecosystem of MRI software, there are two common conventions for representing warping fields:
+
+* **5D composite format (``[x, y, z, t, v]``)**: Originates from Insight Toolkit (ITK), so it also referred to as the ITK format. This of warping field is used by SCT and Advanced Normalization Tools (ANTs). This format is defined by in the "Vector-Valued Datasets" section of the `NIFTI1 Specification <https://nifti.nimh.nih.gov/pub/dist/src/niftilib/nifti1.h>`_.
+* **4D vector format (``[x, y, z, v]``)**: Used by the FMRIB Software Library (FSL) and Statistical Parametric Mapping (SPM) software packages. This format is defined in the "Deformation model" section of the `FSLWiki <https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FNIRT/UserGuide#Deformation_model>`_
+
+For both formats, the ``v`` axis will be of size 3, with each index representing one component of the displacement vector. So, ``v=0`` refers to the ``x`` displacement for each ``[x, y, z]`` coordinate, ``v=1`` refers to the ``y`` displacement, and ``v=2`` refers to the ``z`` displacement.
+
+Compatibility
+=============
+
+SCT generates warping fields in the 5D composite ITK format. This format is incompatible with non-ITK software such as FSL or SPM. But, a conversion between the two formats can be performed using ``sct_image``:
+
+.. code::
+
+   # Split ITK warping field into 3 separate volumes containing X, Y and Z displacements
+   sct_image -i warp_itk.nii.gz -mcs -o warp_vec.nii.gz
+   # Merge into FSL-compatible warping field
+   sct_image -i warp_vec_*.nii.gz -concat t -o warp_fsl.nii.gz
+
+.. warning:: This ITK/FSL warping field conversion only works if the source and destination files are in the same voxel space. If not, there is currently no easy way to do this conversion. The issue is being investigated (more details at: https://github.com/neuropoly/spinalcordtoolbox/issues/2525).
```

Comment:
Right now, my "Warping Fields" page has this warning, which was copied from the SCT course PDF.

#2525 was fixed by #2529, though, which added `sct_image -to-fsl`. So, I presume I should replace the two commands shown here with the new command.

The trouble I'm having is that the usage of the new `-to-fsl` option is quite hard to explain compared to the old commands.

```bash
   # Convert ITK warping field to FSL warping field (when warp_itk and IM_SRC are in the same space)
   sct_image -i warp_itk.nii.gz -to-fsl IM_SRC -o warp_fsl.nii.gz
   # Convert ITK warping field to FSL warping field (when warp_itk and IM_SRC are in different spaces)
   sct_image -i warp_itk.nii.gz -to-fsl IM_SRC (IM_DEST) -o warp_fsl.nii.gz
```

* How will the user know which file to specify for `IM_SRC`?
* How will the user know whether to use the first or the second command?

Before, you only needed the `warp_itk.nii.gz` file... now the process is much more complicated. :sweat_smile:

Reply:
I have tried my best to explain the usage in a597c0e. I referred heavily to the usage examples from the help description.

https://github.com/neuropoly/spinalcordtoolbox/blob/e07cd93cdd9f856b326f81efdd470d10ea642af2/spinalcordtoolbox/scripts/sct_image.py#L139-L145